### PR TITLE
Revert "[sensorfw] fix up standby resume the adaptors"

### PR DESCRIPTION
**Pull request by mikkoharju**

### CODE REVIEW

File: core/hybrisadaptor.cpp [hunk 1]
```diff
@@ -249,23 +241,15 @@ bool HybrisManager::resumeReader(HybrisAdaptor *adaptor)
...
-            qDebug() << Q_FUNC_INFO << list.at(i) << "running";
-            okToStandby = false;
-        }
-    }
-
-    if (okToStandby) {
-        sensordLogD() << "deactivating for standby" << adaptor->name();
+    sensordLogD() << Q_FUNC_INFO << adaptor->id() << adaptor->deviceStandbyOverride();
+    if (!adaptor->deviceStandbyOverride()) {
+        setDelay(adaptor->sensorHandle, 1 * 1000000);
```
Comment (lpotter) [inline core/hybrisadaptor.cpp]: I thought about doing this (and it's a good idea), but we should not presume that all sensors/clients will be happy about slowing down data when screen is blank.
I would rather set the delay for blanking somewhere else, like in the client.

File: core/hybrisadaptor.cpp [hunk 2]
```diff
@@ -226,19 +227,10 @@ void HybrisManager::stopReader(HybrisAdaptor *adaptor)
...
-            okToResume = true;
-        }
-    }
-
-    if (okToResume) {
-        sensordLogD() << "activating for resume" << adaptor->name();
+    sensordLogD() << Q_FUNC_INFO << adaptor->id() << adaptor->deviceStandbyOverride();
+    if (!adaptor->deviceStandbyOverride()) {
+        setDelay(adaptor->sensorHandle, adaptor->interval() * 1000000);
+    } else {
```
Comment (lpotter) [inline core/hybrisadaptor.cpp]: See my comment about standbyOverride. it is a property that tells sensorfw that the client wants to remain running while screen is blanked.

**Comment by lpotter** [comment]
I'd rather not submit this as is. It is using the standbyOverride property in the opposite way that it should.

**Comment by mikkoharju** [comment]
This is more or less blind git blame revert. I added some debug hereabouts, and noticed the `device->activate` is not being called at all. On `standbyReader`, the `device` was deactivated. On `resumeReader` the condition `okToResume` was always false. This revert "fixed" that but very likely broke something else :)

**Comment by lpotter** [comment]
It's been several months.
If whatever this patch is fixing is still a bug please report the bug at https://bugs.merproject.org/

**Pull request closed without merging**